topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -0,0 +1,311 @@
+>
+> > ### {% icon solution %} Solution
+> >
+> > Only the secondary N-methylamino group will be protonated because indoles much like aromatic amides are typically not basic.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Splitting ePharmaLib into individual pharmacophores
+
+The ePharmaLib subset representing human protein targets (*ePharmaLib_PHARAO_human.phar*) is a concatenated file containing 5,010 individual pharmacophore files. In order to speed up our analysis, it is preferable to split the dataset into individual files in order to perform several pharmacophore alignments in parallel, using Galaxy's collection functionality.
+
+> ### {% icon hands_on %} Hands-on 5: Splitting ePharmaLib
+>
+> 1. {% tool [Split file](toolshed.g2.bx.psu.edu/repos/bgruening/split_file_to_collection/split_file_to_collection/0.5.0) %} with the following parameters:
+>    - *"Select the file type to split"*: `Generic`
+>        - {% icon param-file %} *"File to split"*: `output` (Input dataset)
+>        - *"Method to split files"*: `Specify record separator as regular expression`
+>            - *"Regex to match record separator"*: `\$\$\$\$`
+>            - *"Split records before or after the separator?"*: `After`
+>        - *"Specify number of output files or number of records per file?"*: `Number of records per file ('chunk mode')`
+>        - *"Base name for new files in collection"*: `epharmalib`
+>        - *"Method to allocate records to new files"*: `Maintain record order`
+>
+>    > ### {% icon details %} What is a PHAR file?
+>    >
+>    > A PHAR file is essentially a series of lines containing the three-dimensional coordinates of pharmacophoric features and excluded spheres. The first column specifies a feature type (e.g. HACC is a hydrogen bond acceptor). Subsequent columns specify the position of the feature center in a three-dimensional space. Individual pharmacophores are separated by lines containing four dollar signs (*$$$$*).
+>    {: .details}
+>
+{: .hands_on}
+
+## Ligand conformational flexibility
+
+In order to avoid combinatorial explosion, the Align-it ({% cite Taminau2008 %}) tool performs rigid alignment rather than flexible alignment. Conformational flexibility of the ligand is accounted for by introducing a preliminary step, in which a set of energy-minimized conformers for the query ligand are generated with the RDConf ({% cite rdconf %}) tool (using the RDKit ({% cite landrum2013rdkit %}) toolkit). 
+
+> ### {% icon hands_on %} Hands-on 6: Low-energy ligand conformer search
+>
+> 1. {% tool [RDConf: Low-energy ligand conformer search](toolshed.g2.bx.psu.edu/repos/bgruening/rdconf/rdconf/2020.03.4+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `outfile` (output of **Add hydrogen atoms** {% icon tool %})
+>    - *"Maximum number of conformers to generate per molecule"*: `100`
```

Comment:
When I try and run this tool on the input, I only get 2-5 results in the output SD-file. Is this a problem with the tool or am I missing something?

Reply:
Thanks for the interesting question. Staurosporine is a fused 8-ring system with only two rotatable bonds, due to its planar aromatic 5-ring indolocarbozole scaffold which confers the compound a high structural rigidity, i.e. it exists in relatively few energetically distinct 3D conformations.

I equally added your question (and my answer) in the tutorial for the users.